tianshou/policy/modelfree/a2c.py

```diff
@@ -9,42 +10,32 @@
-        :class:`~tianshou.policy.BasePolicy`. (s -> logits)
-    :param torch.nn.Module critic: the critic network. (s -> V(s))
-    :param torch.optim.Optimizer optim: the optimizer for actor and critic network.
+    :param actor: the actor network following the rules in BasePolicy. (s -> logits)
+    :param critic: the critic network. (s -> V(s))
+    :param optim: the optimizer for actor and critic network.
     :param dist_fn: distribution class for computing the action.
-    :param float discount_factor: in [0, 1]. Default to 0.99.
-    :param float vf_coef: weight for value loss. Default to 0.5.
-    :param float ent_coef: weight for entropy loss. Default to 0.01.
-    :param float max_grad_norm: clipping gradients in back propagation. Default to
-        None.
-    :param float gae_lambda: in [0, 1], param for Generalized Advantage Estimation.
-        Default to 0.95.
-    :param bool reward_normalization: normalize estimated values to have std close to
-        1. Default to False.
-    :param int max_batchsize: the maximum size of the batch when computing GAE,
-        depends on the size of available memory and the memory cost of the
-        model; should be as large as possible within the memory constraint.
-        Default to 256.
-    :param bool action_scaling: whether to map actions from range [-1, 1] to range
-        [action_spaces.low, action_spaces.high]. Default to True.
-    :param str action_bound_method: method to bound action to range [-1, 1], can be
-        either "clip" (for simply clipping the action), "tanh" (for applying tanh
-        squashing) for now, or empty string for no bounding. Default to "clip".
-    :param Optional[gym.Space] action_space: env's action space, mandatory if you want
-        to use option "action_scaling" or "action_bound_method". Default to None.
-    :param lr_scheduler: a learning rate scheduler that adjusts the learning rate in
-        optimizer in each policy.update(). Default to None (no lr_scheduler).
-    :param bool deterministic_eval: whether to use deterministic action instead of
-        stochastic action sampled by the policy. Default to False.
+    :param action_space: env's action space
+    :param vf_coef: weight for value loss.
+    :param ent_coef: weight for entropy loss.
+    :param max_grad_norm: clipping gradients in back propagation.
+    :param gae_lambda: in [0, 1], param for Generalized Advantage Estimation.
+    :param max_batchsize: the maximum size of the batch when computing GAE.
+    :param discount_factor: in [0, 1].
+    :param reward_normalization: normalize estimated values to have std close to 1.
+    :param deterministic_eval: if True, use deterministic evaluation.
+    :param observation_space: the space of the observation.
+    :param action_scaling: if True, scale the action from [-1, 1] to the range of
+        action_space. Only used if the action_space is continuous.
+    :param action_bound_method: method to bound action to range [-1, 1].
```

Comment:
missing `Only used if the action_space is continuous.`?

Reply:
Done